Babylon/groups/api/groups/connector/groups/__init__.py

```diff
@@ -0,0 +1,3 @@
+
+list_groups = [
+]
```

Comment:
I can't review an empty file
But for Babylon/templates/working_dir_template/API/API_FILES_HERE I would use a `.gitkeep` file. Matter of taste.

Reply:
Babylon/templates/working_dir_template/API/API_FILES_HERE was an empty file
for nexts empty directory i'ill use .gitkeep

Thank